mem0/memory/main.py

```diff
@@ -179,7 +182,7 @@ def add(
                 self.graph.user_id = "USER"
             added_entities = self.graph.add(data)
 
-        return {"message": "ok"}
```

Comment:
You have to consider the graph memory as well in the return statement

Reply:
added